pyproject.toml

```diff
@@ -27,23 +26,22 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = ">=3.8,<3.12"
 cryptography = "3.3.2"
-napalm = "^3.0"
-lxml = "4.6.5"
+napalm = "4.1.0"
+lxml = "5.2.1"
```

Comment:
I get that this was static before, but should be a bit more loose.

Reply:
Updated